app/src/main/java/me/ivg2/foodapp/HomeActivity.java

```diff
@@ -1,13 +1,51 @@
 package me.ivg2.foodapp;
 
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.design.widget.BottomNavigationView;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.AppCompatActivity;
+import android.view.MenuItem;
 
 public class HomeActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_home);
+
+        final FragmentManager fragmentManager = getSupportFragmentManager();
+
+        final Fragment recipeFragment = new RecipeFragment();
+        final Fragment addFoodFragment = new AddFoodFragment();
+        final Fragment fridgeFragment = new FridgeFragment();
+
+        FragmentTransaction fragmentTransaction1 = fragmentManager.beginTransaction();
```

Comment:
Call this variable `transaction`, there's only one of them so no need to do 1,23,4... etc (also, calling them 1, 2, 3, 4, etc is not super great naming in general.

Reply:
Also, if you're not using the variable, this is equivalent to
```java
fragmentManager
    .beginTransaction()
    .replace(R.id.homeFragment, recipeFragment)
    .commit();
```
which I would argue is more legible.